sourcemap/mapper.go

```diff
@@ -71,3 +82,16 @@ func (m *SmapMapper) NewSourcemapAdded(id Id) {
 	}
 	m.Accessor.Remove(id)
 }
+
+func subSlice(from, to int, content []string) []string {
+	if from > len(content) {
+		return content
```

Comment:
If I read correct, then in this case we would store the whole `content`. I suggest to return an empty array and to log an error.

Reply:
this is for when source content is not available, will rewrite to `if len(content) == 0` to make it clear; sorry for that.
not sure about logging an error, sourcemap content is optional (as per spec); i think some tools include it by default and others don't, users can choose.
you still think should be logged?

Reply:
Oh ok. I agree that there is no need for logging when you check `if len(content) == 0 { return content }`, as this reflects something different than checking that `from` is not an overflow of the content array.